src/native/libs/System.Globalization.Native/pal_localeNumberData.c

```diff
@@ -178,6 +186,11 @@ static int GetNumericPattern(const UNumberFormat* pNumberFormat,
 
     free(icuPattern);
 
+    if (!normalizedPattern)
+    {
+        return INVALID_FORMAT;
```

Comment:
Would `U_MEMORY_ALLOCATION_ERROR` be more appropriate?

Reply:
Fixed. Thank you.